client/express-checkout/event-handlers.js

```diff
@@ -3,69 +3,94 @@
  * External dependencies
  */
 import { __ } from '@wordpress/i18n';
+import { applyFilters } from '@wordpress/hooks';
```

Comment:
All the changes here are coming from `client/tokenized-express-checkout/event-handlers.js`